scanner/build.gradle.kts

```diff
@@ -38,6 +39,7 @@ dependencies {
     implementation(project(":utils"))
 
     implementation("com.fasterxml.jackson.module:jackson-module-kotlin:$jacksonVersion")
+    implementation("com.zaxxer:HikariCP:$hikariVersion")
```

Comment:
Personally, I find it strange to introduce an unused dependency, and only start using it in the next commit. Other reviewers might think differently, but I'd prefer to squash this commit with the next one.

However, this is not a blocker for me.

Reply:
Squashed the commits.